ibridges/__init__.py

```diff
@@ -7,6 +7,13 @@
 from ibridges.session import Session
 from ibridges.tickets import Tickets
 
+try:  # Python < 3.10 (backport)
+    from importlib_metadata import version  # type: ignore
+except ImportError:
+    from importlib.metadata import version  # type: ignore [assignment]
+
+__version__ = release = version("ibridges")
```

Comment:
```suggestion
__version__ = version("ibridges")
```
I don't think release is standard, or is there a specific reason to include it?